memeolist/app/src/main/java/org/aerogear/android/app/memeolist/ui/MemeListActivity.java

```diff
@@ -287,6 +294,36 @@ public void onException(Exception exception) {
                     });
         }
 
+        static void likeAnimation(View view, Meme meme) {
```

Comment:
I do not mind that kind of stuff, but it seems redundant for the actual sync example and there is plenty of code related to it. Can we get that somewhere else so people who will check how example app works will not be bothered by noise of animation etc.

Reply:
Ah.. sorry. By somewhere else can we get that outside activity. Sorry for not being clear.